nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-components/src/main/java/org/apache/nifi/registry/flow/mapping/NiFiRegistryFlowMapper.java

```diff
@@ -567,7 +589,7 @@ private Map<String, String> mapProperties(final ComponentNode component, final C
         return mapped;
     }
 
-    private String encrypt(final String value) {
+    protected String encrypt(final String value) {
```

Comment:
Is there a reason for changing the visibility on this method?

Reply:
I needed to override it in 'FlowAnalysisUtil.createMapper'.
Not trivial why this is necessary. I need an encryptor so that encrypted values cannot be viewed as plain text in a rule. But if I used the original `encrypt` method, it would wrap it in and `enc{}` expression which later causes issues.
With this setup every encrypted value is returned as literal of `"*****"`. Although now that I look at the implementation it may have a bug. But that would be the idea.

Reply:
Thanks for the background, that is understandable. It might be worth add a Java Doc comment summarizing the current method implementation, and also noting the different in the overridden method. Just a sentence highlighting the different in return values, along the lines of your comment, would be helpful.